.changeset/spotty-dogs-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/split-button': patch
+---
+
+Updates Menu. Resolves failing tests due to Menu update
```

Comment:
What were the failing tests due to menu update - will this cause user tests to fail?

Reply:
These were caused by moving to use `onEntered`. 
Overall this will be a major change to `Menu` and part of this epic will be to add test harnesses, so I'm not too concerned about consumer's tests at this point